crewai_tools/tools/selenium_scraping_tool/selenium_scraping_tool.py

```diff
@@ -116,64 +117,60 @@ def _run(
         website_url = kwargs.get("website_url", self.website_url)
         css_element = kwargs.get("css_element", self.css_element)
         return_html = kwargs.get("return_html", self.return_html)
-        driver = self._create_driver(website_url, self.cookie, self.wait_time)
-
-        content = self._get_content(driver, css_element, return_html)
-        driver.close()
-
-        return "\n".join(content)
+        try:
+            self._make_request(website_url, self.cookie, self.wait_time)
+            content = self._get_content(css_element, return_html)
+            return "\n".join(content)
+        finally:
+            self.driver.close()
 
-    def _get_content(self, driver, css_element, return_html):
+    def _get_content(self, css_element, return_html):
         content = []
 
         if self._is_css_element_empty(css_element):
-            content.append(self._get_body_content(driver, return_html))
+            content.append(self._get_body_content(return_html))
         else:
-            content.extend(self._get_elements_content(driver, css_element, return_html))
+            content.extend(self._get_elements_content(css_element, return_html))
 
         return content
 
     def _is_css_element_empty(self, css_element):
         return css_element is None or css_element.strip() == ""
 
-    def _get_body_content(self, driver, return_html):
-        body_element = driver.find_element(self._by.TAG_NAME, "body")
+    def _get_body_content(self, return_html):
+        body_element = self.driver.find_element(self._by.TAG_NAME, "body")
 
         return (
             body_element.get_attribute("outerHTML")
             if return_html
             else body_element.text
         )
 
-    def _get_elements_content(self, driver, css_element, return_html):
+    def _get_elements_content(self, css_element, return_html):
         elements_content = []
 
-        for element in driver.find_elements(self._by.CSS_SELECTOR, css_element):
+        for element in self.driver.find_elements(self._by.CSS_SELECTOR, css_element):
             elements_content.append(
                 element.get_attribute("outerHTML") if return_html else element.text
             )
 
         return elements_content
 
-    def _create_driver(self, url, cookie, wait_time):
+    def _make_request(self, url, cookie, wait_time):
         if not url:
             raise ValueError("URL cannot be empty")
 
         # Validate URL format
         if not re.match(r"^https?://", url):
             raise ValueError("URL must start with http:// or https://")
 
-        options = self._options
-        options.add_argument("--headless")
-        driver = self.driver(options=options)
```

Comment:
we were trying to access driver as a callable, but this object is not callable.. 
I moved the driver creation to `init` applying their proper options